lib/synsbasen_api/resources/version.rb

```diff
@@ -20,7 +20,13 @@ def find(id, expand: [])
       # @return [ApiResponse] An instance of `ApiResponse` containing details
       #   of all versions associated with the specified variant.
       def all(variant_id, expand: [])
-        get("/v1/variants/#{variant_id}/versions", expand: expand)
+        get("/v1/#{resource_name}/#{variant_id}/versions", expand: expand)
```

Comment:
This doesn't seem correct - /versions should be on the /variants/:id/versions endpoint

```suggestion
        get("/v1/variants/#{variant_id}/#{resource_name}", expand: expand)
```